MonologBundle.php

```diff
@@ -38,6 +41,10 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new FixEmptyLoggerPass($channelPass));
         $container->addCompilerPass(new AddProcessorsPass());
         $container->addCompilerPass(new AddSwiftMailerTransportPass());
+
+        if (class_exists(MessageBusInterface::class) && class_exists(ResetLoggersWorkerSubscriber::class)) {
```

Comment:
Only ResetLoggersWorkerSubscribed is needed, right?

Reply:
No, both are needed to me. `ResetLoggersWorkerSubscriber` is from MonologBridge. If Monolog bridge is installed but not Messenger, we don't want `ResetLoggersWorkerSubscriber` to be activated.

Reply:
Of course, thanks.